### PR TITLE
Psychic link is manually cancellable, Buffs range

### PR DESCRIPTION
**Pull request by Walarks**
## About The Pull Request

Psychic link process is now able to be cancelled manually by activating the skill again.
Also increases psychic link range.
## Why It's Good For The Game

Psychic link seems underused in most cases, so I'm interested in buffing its range to make it easier to utilize. I think this would still keep it somewhat proactive by requiring some setup as opposed to making it more reactive by reducing the channeling time.
## Changelog
:cl:
qol: psychic link's activation can be manually cancelled by activating the skill again
balance: psychic link has 15 instead of 10 range
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
Cancellable seems okay, range seems like a creep.

**Comment by Walarks** [comment]
> 

The thing is that it almost isn't used currently. Also when you are linked as gorger, you don't bring anything else to a fight.

**Comment by goose4429** [comment]
It is 100% used in RUTGMC, and has good performance. Cancellation is nice, buff range is too much. Although +1 or +2 tiles seems like what has to be done.

**Comment by Walarks** [comment]
> It is 100% used in RUTGMC, and has good performance. Cancellation is nice, buff range is too much. Although +1 or +2 tiles seems like what has to be done.

I made this change based on my experience on TGMC instead of RUTGMC. I don't think RUTGMC should be a concern since it's a different server.

File: code/datums/status_effects/xeno_buffs.dm [hunk 1]
```diff
@@ -101,7 +101,7 @@
 	///Minimum health threshold before the effect is deactivated
 	var/minimum_health
 	///If the target xeno was within range
-	var/was_within_range = TRUE
+	var/was_within_range
```
Comment (TiviPlus) [inline code/datums/status_effects/xeno_buffs.dm:104]: is this a bool or not cus itf its a bool make it false explicitly

**Review by TiviPlus — APPROVED** [review]
rest lgtm

File: code/datums/status_effects/xeno_buffs.dm (hunk 1 above)
Comment (Walarks) [reply code/datums/status_effects/xeno_buffs.dm]: updated

**Pull request merged**